olp-cpp-sdk-dataservice-read/src/repositories/PartitionsRepository.cpp

```diff
@@ -38,13 +40,64 @@ namespace repository {
 using namespace olp::client;
 
 namespace {
+constexpr char kPRLogtag[] = "DataRepository";
```

Comment:
Shouldn't it be `PartitionsRepository`?
```suggestion
constexpr char kPRLogtag[] = "PartitionsRepository";
```